pkg/clusterd/disk.go

```diff
@@ -41,7 +42,7 @@ func ignoreDevice(d string) bool {
 	return isRBD.MatchString(d)
 }
 
-// Discover all the details of devices available on the local node
+// DiscoverDevices all the details of devices available on the local node
```

Comment:
More grammatical completion to sentence?

```suggestion
// DiscoverDevices returns all the details of devices available on the local node
```